diff -rupw ncurses-6.4-orig/misc/run_tic.in ncurses-6.4/misc/run_tic.in
--- ncurses-6.4-orig/misc/run_tic.in	2020-02-12 17:50:22.000000000 +0800
+++ ncurses-6.4/misc/run_tic.in	2020-12-30 10:26:00.107716122 +0800
@@ -40,13 +40,13 @@
 echo "** Building terminfo database, please wait..."
 #
 # The script is designed to be run from the misc/Makefile as
 #	make install.data
 
 : "${suffix:=@EXEEXT@}"
-: "${DESTDIR:=@DESTDIR@}"
+#: "${DESTDIR:=@DESTDIR@}"
 : "${prefix:=@prefix@}"
 : "${exec_prefix:=@exec_prefix@}"
 : "${bindir:=@bindir@}"
 : "${top_srcdir:=@top_srcdir@}"
 : "${srcdir:=@srcdir@}"
 : "${datarootdir:=@datarootdir@}"
